README.md

```diff
@@ -8,12 +8,13 @@ Czech Python community homepage.
 
 ## Installation
 
-The code is **Python 3** (the production Python version is 3.6).
+The code is **Python 3.7**.
 
 ```sh
+$ pip install pipenv
```

Comment:
This will be a problem for most of the beginners who want to get involved. Link to some how to install pipenv instead please.

Reply:
Huh? I always install pipenv like this 🤔 But it seems [it's not as simple for everyone](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv). I'll add something to the README.

Reply:
```
$ pip install pipenv
Collecting pipenv
  Using cached https://files.pythonhosted.org/packages/bb/15/b155a5c0d19ce41609f50bb70a37e0de092b453ec4bd2eac59e53a2c3227/pipenv-2018.11.26-py2-none-any.whl
Requirement already satisfied: setuptools>=36.2.1 in /usr/lib/python2.7/site-packages (from pipenv) (40.4.3)
Requirement already satisfied: virtualenv in /usr/lib/python2.7/site-packages (from pipenv) (16.0.0)
Requirement already satisfied: enum34; python_version < "3" in /usr/lib/python2.7/site-packages (from pipenv) (1.1.6)
Requirement already satisfied: pip>=9.0.1 in /usr/lib/python2.7/site-packages (from pipenv) (18.1)
Collecting typing; python_version < "3.5" (from pipenv)
  Using cached https://files.pythonhosted.org/packages/cc/3e/29f92b7aeda5b078c86d14f550bf85cff809042e3429ace7af6193c3bc9f/typing-3.6.6-py2-none-any.whl
Collecting certifi (from pipenv)
  Using cached https://files.pythonhosted.org/packages/9f/e0/accfc1b56b57e9750eba272e24c4dddeac86852c2bebd1236674d7887e8a/certifi-2018.11.29-py2.py3-none-any.whl
Collecting virtualenv-clone>=0.2.5 (from pipenv)
  Using cached https://files.pythonhosted.org/packages/16/9d/6419a4f0fe4350db7fdc01e9d22e949779b6f2d2650e4884aa8aededc5ae/virtualenv_clone-0.4.0-py2.py3-none-any.whl
Installing collected packages: typing, certifi, virtualenv-clone, pipenv
Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/lib/python2.7/site-packages/typing.pyc'
Consider using the `--user` option or check the permissions.
```

Reply:
```
$ pip3 install pipenv
Collecting pipenv
  Downloading https://files.pythonhosted.org/packages/13/b4/3ffa55f77161cff9a5220f162670f7c5eb00df52e00939e203f601b0f579/pipenv-2018.11.26-py3-none-any.whl (5.2MB)
    100% |████████████████████████████████| 5.2MB 360kB/s
Requirement already satisfied: certifi in /usr/local/lib/python3.7/site-packages (from pipenv) (2018.4.16)
Requirement already satisfied: virtualenv in /usr/local/lib/python3.7/site-packages (from pipenv) (16.0.0)
Requirement already satisfied: setuptools>=36.2.1 in /usr/local/lib/python3.7/site-packages (from pipenv) (40.5.0)
Requirement already satisfied: pip>=9.0.1 in /usr/local/lib/python3.7/site-packages (from pipenv) (18.1)
Requirement already satisfied: virtualenv-clone>=0.2.5 in /usr/local/lib/python3.7/site-packages (from pipenv) (0.3.0)
Installing collected packages: pipenv
Successfully installed pipenv-2018.11.26
```

👋 🍎

Reply:
pip3 is not pip